src/generated/resources/data/gtceu/tags/items/electric_pumps.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_electric_pump",
+    "gtceu:mv_electric_pump",
+    "gtceu:hv_electric_pump",
+    "gtceu:ev_electric_pump",
+    "gtceu:iv_electric_pump",
+    "gtceu:luv_electric_pump",
+    "gtceu:zpm_electric_pump",
+    "gtceu:uv_electric_pump",
+    "gtceu:uhv_electric_pump",
+    "gtceu:uev_electric_pump",
+    "gtceu:uiv_electric_pump",
+    "gtceu:uxv_electric_pump",
+    "gtceu:opv_electric_pump"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.